### DISCUSSION TITLE
Welcome to Canvas-Editor Discussions!

### DISCUSSION THREAD

**Discussion by vishanurag** [vishanurag/Canvas-Editor#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DevROHIT11** [comment]
hello Sir ,
I really like your website and the content also , i want to contribute in your project make the project  better !!

**Reply by Akash-Gupta-git** [reply]
I also really impress by your website 
mouse moment 
but In your website has silly mistake  
overlaping text
![image](https://github.com/user-attachments/assets/870e245e-5c78-4491-92ad-12228fc8e9bb)

**Comment by DevROHIT11** [comment]
@vishanurag i had resolved the merge conflict of my pr #796 can u plz accept my request
